### PR TITLE
Update .github/CODEOWNERS

### PR DESCRIPTION
**Pull request by mik-laj**
I am not interested to maintain this provider package. I started a new job and now have less time to work in the community.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -26,8 +26,8 @@
 /airflow/www/ @ryanahamilton @ashb
 
 # Providers
-/airflow/providers/google/ @turbaszek @mik-laj
-/airflow/providers/snowflake @turbaszek @potiuk
+/airflow/providers/google/ @turbaszek 
+/airflow/providers/snowflake/ @turbaszek @potiuk
```
Comment (potiuk) [inline .github/CODEOWNERS]:
```suggestion
/airflow/providers/snowflake/ @turbaszek @potiuk @mik-laj 
```

Comment (mik-laj) [reply .github/CODEOWNERS]: Done

**Review by kaxil — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -26,8 +26,8 @@
 /airflow/www/ @ryanahamilton @ashb
 
 # Providers
-/airflow/providers/google/ @turbaszek @mik-laj
-/airflow/providers/snowflake @turbaszek @potiuk
+/airflow/providers/google/ @turbaszek 
+/airflow/providers/snowflake/ @turbaszek @potiuk @mik-laj
```
Comment (potiuk) [inline .github/CODEOWNERS]:
```suggestion
/airflow/providers/snowflake/ @turbaszek @potiuk @mik-laj
```

**Review by potiuk — APPROVED** [review]

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -26,8 +26,8 @@
 /airflow/www/ @ryanahamilton @ashb
 
 # Providers
-/airflow/providers/google/ @turbaszek @mik-laj
-/airflow/providers/snowflake @turbaszek @potiuk
+/airflow/providers/google/ @turbaszek
```
Comment (mik-laj) [inline .github/CODEOWNERS]:
```suggestion
/airflow/providers/google/ @turbaszek
```

**Pull request merged**